examples/datasource-http/README.md

```diff
@@ -1,3 +1,62 @@
-# Grafana Data Source Plugin HTTP example
+# Basic Data Source HTTP Plugin for Grafana
 
-This example queries data from an HTTP API.
+This repository contains a data source plugin that uses HTTP through Grafana's data source HTTP proxy.
+## Overview
+
+The Grafana HTTP Backend Data Source Plugin shows the integration of a backend HTTP service as a custom data source within Grafana. This plugin serves as a reference implementation for developers seeking to incorporate HTTP-based data sources into their Grafana dashboards.
```

Comment:
This is the frontend only datasource - maybe copy paste error?

Reply:
done